### ISSUE TITLE
Help block on how to restore from earlier backups

### ISSUE THREAD

**Issue by ain** [tmux-plugins/tmux-resurrect#99]
Would be nice to have a section in the README on how to restore the earlier dump in case someone accidentally saves unrestored (empty) tmux status. 

Just happened to me so I manually symlinked the older dump.

**Comment by bruno-** [comment]
Hey,
this is a good idea. Wanna open a PR for this?

I documented this in [tmux-continuum readme](https://github.com/tmux-plugins/tmux-continuum#faq)
but it's definitely a good idea to have this in tmux-resurrect as well.

**Comment by ain** [comment]
Sure thing, will do.

**Comment by Strangehill** [comment]
yes! I looked around for instructions on restoring from a specific backup, and that link to the continuum documentation was how i finally figured it out.
https://github.com/tmux-plugins/tmux-continuum/blob/master/docs/faq.md
